actors/runtime/src/main/java/cloud/orbit/actors/concurrent/WaitFreeExecutionSerializer.java

```diff
@@ -74,9 +74,9 @@ public WaitFreeExecutionSerializer(final ExecutorService executorService, Object
         final Task<R> completion = new Task<>();
 
         int queueSize = size.get();
-        if (DEBUG_ENABLED && queueSize >= maxQueueSize / 10)
+        if (WARN_ENABLED && queueSize == maxQueueSize / 10)
```

Comment:
I'm not sure this is really a warning. Would it make sense to make this info, and then introduce a warning if you exceed the maxSize?